docs/modules/ROOT/pages/how-tos/configuring/component-nudges.adoc

```diff
@@ -1 +1,87 @@
-= Defining component relationships
\ No newline at end of file
+= Defining component relationships
+
+When an application includes multiple components, their repositories might contain references to image builds of another component. For instance, an application might include multiple child components that, using a Dockerfile `FROM` clause, reference the image build of a common parent.
+
+In such instances, whenever you build a new image of the common parent component, you need to update the references to it. Specifically, you need to copy and paste the pullspec and image digest of that new image to your other source repositories. This process is tedious and error-prone.
+
+However, if you build an application with {ProductName} and make references between components with image digests (`sha256:...`), {ProductName} can automatically generate pull requests to update those references. To use this functionality, simply define the relationships between your components, either in the {ProductName} UI, or in your CLI.
+
+[NOTE]
+====
+By default, {ProductName} only looks for image digest references in Dockerfiles, Containerfiles, and YAML files. However, you can xref:./customizing-the-build.adoc[customize your build pipeline] to adjust that list of files with an annotation. Add the following annotation under `annotations` in your component's `/.tekton/<component-name>-push.yaml` file:
+`build.appstudio.openshift.io/build-nudge-files: "<regex for file type, like .*.md>"`
```

Comment:
I think that these can technically be regex or glob patterns, but it's fine as it is: https://docs.renovatebot.com/string-pattern-matching/ ... I think we can also say a comma separated list of regexes?

Do you want to put the _default_ regex in here so that users can see how the pattern might look?

Reply:
I revised the note to include glob patterns and provide the list of existing regex patterns as examples. Please LMK if we're good to go, or need to add anything else.

Reply:
Did you push the revision? I don't see a change.

Reply:
Yeah I pushed the revision; I can see it here: https://github.com/konflux-ci/docs/blob/Chr1st1anSears-patch-6/docs/modules/ROOT/pages/how-tos/configuring/component-nudges.adoc

Reply:
If you look at lines 11-14 (the whole Note) I think they now address your original point about directing where Konflux looks for image references